src/main/scala/chisel3/tester/package.scala

```diff
@@ -173,5 +173,7 @@ package object tester {
 
   implicit def decoupledToDriver[T <: Data](x: DecoupledIO[T]) = new DecoupledDriver(x)
 
+  implicit def irrevocableToDriver[T <: Data](x: IrrevocableIO[T]) = new DecoupledDriver(x)
```

Comment:
Will an implicit conversion from ReadyValidIIO instead of DecoupledIO work instead of requiring an implicit conversion for each individually?

Reply:
oh yes, it is much better to convert from ReadyValidIO.  
Fixed in 49d2857.